pkg/minikube/constants/constants.go

```diff
@@ -154,38 +107,17 @@ var OldestKubernetesVersion = "v1.11.10"
 var ConfigFile = localpath.MakeMiniPath("config", "config.json")
```

Comment:
Idea for a follow-up PR: this should be moved to `localpath`, and turned into a function. As something that is dynamically generated and respects environment variables,  it's not really a function.

Reply:
done